libafl/src/mutators/multi.rs

```diff
@@ -34,22 +37,24 @@ use crate::{
 /// at once.
 pub trait DefaultMultipartMutator {}
 
-impl<I, M, S> Mutator<MultipartInput<I>, S> for M
+impl<I, M, N, S> Mutator<MultipartInput<I, N>, S> for M
 where
     M: DefaultMultipartMutator + Mutator<I, S>,
     S: HasRand,
 {
     fn mutate(
         &mut self,
         state: &mut S,
-        input: &mut MultipartInput<I>,
+        input: &mut MultipartInput<I, N>,
```

Comment:
I think we can remove the Generic again, just make the guy `CoW<'static, str>` or something

Reply:
The original idea was to use MutlipartInput and have ListInput be a MultipartInput with `()` as name, but you basically did the exact opposite :P

Reply:
Names are just a form of id for each part. So if you don't need it human readable, numbers would probably be slightly more performant, because you don't need to compare the entire string, no?

Reply:
Then should it be `key` instead of `name`? (I'm not sure)

Reply:
I was searching for a better name, but ID seemed weird. I like key.